wandb/sdk/wandb_login.py

```diff
@@ -115,22 +121,32 @@ def __init__(self):
         self._key = None
         self._relogin = None
 
-    def setup(self, kwargs):
-        self.kwargs = kwargs
+    def setup(
+        self,
+        *,
+        anonymous: Optional[Literal["must", "allow", "never"]] = None,
```

Comment:
https://github.com/wandb/wandb/blob/main/wandb/sdk/wandb_settings.py#L1023 interesting that here we validate "true" and "false" as well, but docs don't list these...

Reply:
https://docs.wandb.ai/ref/python/login

Reply:
hmm yeah.. I wonder if people use it. In this PR I'd like to preserve the current behavior.

Reply:
agreed! also "true", "false" seems confusing... (if we ever had that i would depreciate it)